src/openproduct/producttypen/serializers/content.py

```diff
@@ -66,7 +81,15 @@ def get_taal(self, obj):
 
     class Meta:
         model = ContentElement
-        fields = ("uuid", "content", "labels", "producttype_uuid", "taal")
+        fields = (
+            "uuid",
+            "naam",
+            "content",
+            "labels",
+            "labels_detail",
```

Comment:
Is this serializer actually exposed via an API endpoint? I don't see `labels` included in the response for a GET on `/content/{uuid}`

<img width="514" height="468" alt="image" src="https://github.com/user-attachments/assets/230518ba-292c-474e-a798-63c52ab08b7f" />